doc/source/dev_guide/custom_reader.rst

```diff
@@ -60,17 +60,24 @@ if needed (ex. goes-imager).
 :file format: If the file format of the files is informative to the user or
     can distinguish one reader from another then this field should be
     specified. Common format names should be abbreviated following existing
-    abbreviations like `nc` for NetCDF3 or NetCDF4, `hdf` for HDF4, `h5` for
+    abbreviations like ``nc`` for NetCDF3 or NetCDF4, ``hdf`` for HDF4, ``h5`` for
     HDF5.
 
 The existing :ref:`reader's table <reader_table>` can be used for reference.
-When in doubt, reader names can be discussed in the github pull
-request when this reader is added to Satpy or a github issue.
+When in doubt, reader names can be discussed in the GitHub pull
+request when this reader is added to Satpy, or in a GitHub issue.
 
 The YAML file
 -------------
 
-The yaml file is composed of three sections:
+If your reader is going to be part of Satpy, the YAML file should be
+located in the ``satpy/etc/readers`` directory, along with the YAML
+files for all other readers.  If you are developing a reader for internal
+purposes (such as for unpublished data), the YAML file should be located
+in any directory in ``$SATPY_CONFIG_PATH`` within the subdirectory
+``readers/`` (see :ref:`config`).
```

Comment:
You should be able to do:

```
:doc:`../../config`
```

and remove the reference anchor you added to config.rst.